src/programs/state/massachusetts/template.js

```diff
@@ -17,10 +17,10 @@
 // import { SOME_DATA } from '../../../data/state/massachusetts/2018/SOME_DATA.js';
 
 // UTILITIES
-// import { roundMoney } from '../../../helpers/math';
+// import { roundMoney } from '../../../utils/math';
```

Comment:
Just fyi, we'll probably be able to remove this after the client form values stuff has been worked out. Then again, this template would do well to be re-worked. For that matter, I've been debating just having a templates folder. Some of the same questions as the location of the test files come up. On the third (fourth?) hand, there may be little point to templates at this juncture unless we're trying to help our possible future collaborators (or explore some kind of volunteer collaboration format/architecture).

Reply:
Okay - what I'm hearing re: this PR is change nothing. If there's a work item here, I'd suggest populating a separate issue.

Reply:
Good idea.